pages/templatetags/pages_tags.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+from pages.models import *
+
+register = template.Library()
+
+# Notification Banner snippets that are set to show
+@register.inclusion_tag('pages/tags/notification-banner.html', takes_context=True)
+def notification_banner(context):
+    return {
+        'notifications': NotificationBanner.objects.filter(show=True),
```

Comment:
Could we make this get the latest notification banner? Perhaps with https://docs.djangoproject.com/en/1.8/ref/models/querysets/#latest

I want to avoid someone adding two banners on accident if we could, or only allow creation of one notification banner.

Reply:
Ok sure, will just take first record from list then. So as long as 0 or 1 record is set to show then will work as expected. And if more than 1 is set to show will still only display 1.